pkg/pillar/cmd/zedrouter/patchenvelopes.go

```diff
@@ -366,6 +366,7 @@ func (pes *PatchEnvelopes) EnvelopesInUsage() []string {
 // disappear on zedrouter
 type peInfoToDisplay struct {
 	PatchID     string
+	Version     string
```

Comment:
Shouldn't there be an exported struct or a protobuf definiton that apps will import and use to unmarshal data received from metadata server? Or is this already in EVE API repo?

Reply:
This is already a part of EVE API repo [here](https://github.com/lf-edge/eve-api/blob/d4c463c8b9807e4002fee9983be9fafa1a67feb0/proto/config/patch_envelope.proto#L71)